crates/ruff/src/autofix/actions.rs

```diff
@@ -51,28 +51,28 @@ fn is_lone_child(child: &Stmt, parent: &Stmt, deleted: &[&Stmt]) -> Result<bool>
                 bail!("Unable to find child in parent body")
             }
         }
-        StmtKind::Try(ast::StmtTry {
+        Stmt::Try(ast::StmtTry {
             body,
             handlers,
             orelse,
             finalbody,
+            range: _,
         })
-        | StmtKind::TryStar(ast::StmtTryStar {
+        | Stmt::TryStar(ast::StmtTryStar {
             body,
             handlers,
             orelse,
             finalbody,
+            range: _,
```

Comment:
I tend to bias towards `..`, do you prefer targeted ignores for a specific reason?

Reply:
It depends. Targeted ignores are useful in code paths where we want to handle all fields (and get errors, if a new field has been added). Now, reading all the code to know if we should handle all fields was too much work, so I opted for targeted ignores (safer).